gnome-screenlock-password-opensuse132-20160620.json

```diff
@@ -0,0 +1,15 @@
+{
+  "area": [
+    {
+      "type": "match",
+      "ypos": 237,
+      "height": 263,
+      "width": 458,
+      "xpos": 290
+    }
+  ],
+  "tags": [
+    "gnome-screenlock-password"

```

Comment:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/products/sle/main.pm#L107 - something like that does not exist in opensuse/main.pm (but it should)